### DISCUSSION TITLE
Oil Panting Shader/Filter

### DISCUSSION THREAD

**Discussion by caostar** [pixijs/pixijs#8389] [Q&A]
Hi there,

I am trying to re-create the classic photoshop "Oil Painting Filter" and applying it to my whole scene.
But I cant find the proper shader to do the job (and default filters do not include that). Anyone has tried that before?

**Comment by miltoncandelero** [comment]
Something like these?

https://www.shadertoy.com/results?query=oil+paint

**Comment by paint-by-numbers-kits** [comment]
Hey there! Recreating the classic Photoshop "Oil Painting Filter" is indeed an intriguing endeavor. While I don't have a ready-made shader for this task, I'd suggest looking into shader programming libraries or frameworks such as ShaderGraph in Unity or GLSL. They offer a robust platform for creating and experimenting with custom shaders.

Furthermore, you might find inspiration in traditional art techniques. For instance, studying how [NumeralPaint](https://numeralpaint.com/) kits create oil painting-like textures using structured, numerical painting might offer some insights into how you could simulate such an effect digitally.

Additionally, there are communities and forums dedicated to shader programming and digital art creation where you might find pre-existing solutions or expertise on recreating this particular filter. Websites like Shadertoy or forums on Stack Overflow could be beneficial.

I hope this helps
